configure.php

```diff
@@ -424,7 +422,7 @@ function install_plugin( array $plugin_data, bool $prompt, &$installed_plugins )
 	'CREATE_WORDPRESS_PROJECT'     => strtoupper( str_replace( '-', '_', $project_name_slug ) ),
 
 	'vendor_name'                  => $vendor_name,
-	'alleyinteractive'             => $vendor_slug,
```

Comment:
Removing this would make something like this line https://github.com/alleyinteractive/create-wordpress-project/blob/develop/composer.json#L2 not updated, right?

Reply:
True. If we have more of those hardcoded that we need to keep I think we would want to be a lot more specific about the replacements. Something like replacing the whole thing with `$author_username . '/' . $project_slug`